src/features/user/user-slice.jsx

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const initialState = {};
```

Comment:
Nije bas dobra ideja da imas prazan `initialState`. Jer ovako ne moze da se vidi eksplicitno sta ovaj slice nosi od podataka. To je jedna stvar, a druga stvar zbog koje je to lose je sto moras non stop da proveravas da li je ono sto ocekujes u state-u `undefined`.
```
const initialState = {
    current: {}
};
```
Nazvao bih to `current` ili tako nesto, a ne `user` da bih izbegao da `...getState().user.user...`.

Reply:
Sredjeno, napravio sam `current` sa poljima `token` i `data`.